robotiq_hande_driver/include/communication.hpp

```diff
@@ -0,0 +1,127 @@
+#ifndef COMMUNICATION_HPP_
+#define COMMUNICATION_HPP_
+
+#include <stdint.h>
+#include <stddef.h>
+#include <vector>
+#include <modbus/modbus.h>
+
+/**
+ * @brief This class contains low level gripper commands and status
+ */
+
+namespace hande_driver
+{
+constexpr auto kRegisterWordLength = 3;
+enum OutputBytes{
+    OUTPUT_BYTES_RESERVED_1,
```

Comment:
I propose to use the scoped enums, i.e. `enum class Name : type {}` , instead of the plain `enum` to enhance safety.

https://stackoverflow.com/a/18338930

Also, let's set the first enum item to make it absolutely clear.

```suggestion
enum class OutputBytes : uint8_t {
    OUTPUT_BYTES_RESERVED_1 = 0u,
```

Reply:
Makes a lot of sense, adding the suggestion